gst/nnstreamer/tensor_filter/tensor_filter.c

```diff
@@ -169,6 +169,23 @@ static gboolean gst_tensor_filter_sink_event (GstBaseTransform * trans,
       } \
     } while (0)
 
+/**
+ * @brief Check validity of path for given model file if check_model_path_in_nns is set
+ */
+#define gst_tensor_filter_check_model_path_if_necessary(priv,prop,ret) do { \
+  ret = 0; \
+  if (priv->fw->check_model_path_in_nns == TRUE) { \
+    int idx; \
+    for (idx = 0; idx < prop->num_models; idx++) { \
+      if (!g_file_test (prop->model_files[idx], G_FILE_TEST_IS_REGULAR)) { \
```

Comment:
Minor suggestion: how about strictly checking `model_files` like `prop->model_files != NULL`.
Otherwise, it might have potential dangers for a segmentation fault.

Reply:
At the point when the sub-plugin is loaded, prop->num_models is set to 0 by gst_tensor_filter_init. Although adding a null checking is not harmful at all, this seems to be a guard for the segmentation fault that would hardly happen.

Reply:
Sure, it's safe for normal users using a standard way to use `PROP_MODEL`. But, IMO, `.num_models` can be specified from a user (like some unittests), and someone may set this value without assigning `.model_files`. Of course, it's just my personal concerns :)

Reply:
@dongju-chae OK:) I will ease your concerns. Thanks!